pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -576,17 +647,23 @@ func newKubeVirt(expectedNumOutdated int) *v1.KubeVirt {
 	}
 }
 
-func newVirtualMachine(name string, isMigratable bool, image string, vmiSource *framework.FakeControllerSource, podSource *framework.FakeControllerSource) *v1.VirtualMachineInstance {
-	vmi := api.NewMinimalVMI("testvm")
-	vmi.Name = name
-	vmi.Namespace = k8sv1.NamespaceDefault
-	vmi.Status.LauncherContainerImageVersion = image
-	vmi.Status.Phase = v1.Running
+func newVirtualMachineInstance(name string, isMigratable bool, image string) *v1.VirtualMachineInstance {
+	vmi := libvmi.New(
+		libvmi.WithResourceMemory("8192Ki"),
+		libvmi.WithNamespace(k8sv1.NamespaceDefault),
+		libvmi.WithName(name),
+		libvmistatus.WithPhase(v1.Running),
+		libvmistatus.WithLauncherContainerImageVersion(image),
+	)
 	vmi.UID = "1234"
 	if isMigratable {
-		vmi.Status.Conditions = []v1.VirtualMachineInstanceCondition{{Type: v1.VirtualMachineInstanceIsMigratable, Status: k8sv1.ConditionTrue}}
+		libvmistatus.WithCondition(v1.VirtualMachineInstanceCondition{Type: v1.VirtualMachineInstanceIsMigratable, Status: k8sv1.ConditionTrue})(vmi)
```

Comment:
In other tests you create an `options` slice that you add at the end of `libvmi.New()`, I think it would look cleaner here as well.

Reply:
Right!